src/main/java/frc/robot/subsystems/ClawSubsystem/ClawSubsystem.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems.ClawSubsystem;
+
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import edu.wpi.first.wpilibj.Solenoid;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+public class ClawSubsystem extends SubsystemBase{
+    private TalonSRX motor1;
+    private Solenoid pfft1;
+
+    public ClawSubsystem() {
+        
+      CommandScheduler.getInstance().registerSubsystem(this);
```

Comment:
this is a line in the super constructor, which according to Kevin, is already called. So I think we can get away with not having this line